test/common/verify_3scale_UIBBT_alerts.go

```diff
@@ -0,0 +1,192 @@
+package common
+
+import (
+	"context"
+	goctx "context"
+	"fmt"
+	openshiftappsv1 "github.com/openshift/api/apps/v1"
+	k8sappsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/json"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/util/retry"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	"strings"
+	"time"
+)
+
+type Alert struct {
+	Labels struct {
+		AlertName string `json:"alertname"`
+	} `json:"labels"`
+	State string `json:"state"`
+}
+
+type AlertsResults struct {
+	Alerts []Alert `json:"alerts"`
+}
+
+var cfg *rest.Config
+var testEnv *envtest.Environment
+
+func TestThreeScaleUIBBTAlerts(t TestingTB, ctx *TestingContext) {
+
+	// Get the kube client and namespace from the shared test context
+	client := ctx.Client
+	// start test env
+	useCluster := true
+	testEnv = &envtest.Environment{
+		UseExistingCluster:       &useCluster,
+		AttachControlPlaneOutput: true,
+	}
+	var err error
+	cfg, err = testEnv.Start()
+	if err != nil {
+		t.Fatalf("could not get start test environment %s", err)
+	}
+
+	installationType, err := GetInstallType(cfg)
+	fmt.Println(installationType)
+	if err != nil {
+		t.Logf("Failed to identify installation type: %v", err)
+		t.FailNow()
+	}
+
+	var namespaceDeployment string
+	var namespaceConfig string
+
+	if installationType == "managed-api" {
```

Comment:
Yes, the namespace names should not be hard coded like this, NamespacePrefix var needs to be used:
https://github.com/integr8ly/integreatly-operator/blob/master/test/common/types.go#L19

If you need to change logic based on installationType, use the existing vars instead of hard-coded strings as in following snippet:
https://github.com/integr8ly/integreatly-operator/blob/master/controllers/rhmi/types.go#L133-L159

Definition of the vars:
https://github.com/integr8ly/integreatly-operator/blob/master/apis/v1alpha1/rhmi_types.go#L44